[2025-10-03T12:12:57.081+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T12:12:57.121+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_one.hello scheduled__2025-10-03T12:11:00+00:00 [queued]>
[2025-10-03T12:12:57.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_one.hello scheduled__2025-10-03T12:11:00+00:00 [queued]>
[2025-10-03T12:12:57.136+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-03T12:12:57.156+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello> on 2025-10-03 12:11:00+00:00
[2025-10-03T12:12:57.169+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=48) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T12:12:57.172+0000] {standard_task_runner.py:63} INFO - Started process 50 to run task
[2025-10-03T12:12:57.171+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_one', 'hello', 'scheduled__2025-10-03T12:11:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/dag_one.py', '--cfg-path', '/tmp/tmpgwyinci2']
[2025-10-03T12:12:57.176+0000] {standard_task_runner.py:91} INFO - Job 2: Subtask hello
[2025-10-03T12:12:57.199+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-03T12:12:57.294+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_one.hello scheduled__2025-10-03T12:11:00+00:00 [running]> on host 7aff8d0a020c
[2025-10-03T12:12:57.582+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_one' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T12:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-03T12:11:00+00:00'
[2025-10-03T12:12:57.590+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T12:12:57.592+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-03T12:12:57.597+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "My name is Ali Moahmed and I am a data engineer and my age is 21. Wellcome to Airflow"']
[2025-10-03T12:12:57.645+0000] {subprocess.py:86} INFO - Output:
[2025-10-03T12:12:57.669+0000] {subprocess.py:93} INFO - My name is Ali Moahmed and I am a data engineer and my age is 21. Wellcome to Airflow
[2025-10-03T12:12:57.675+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-03T12:12:57.679+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T12:12:57.753+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_one, task_id=hello, run_id=scheduled__2025-10-03T12:11:00+00:00, execution_date=20251003T121100, start_date=20251003T121257, end_date=20251003T121257
[2025-10-03T12:12:57.832+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-03T12:12:57.866+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-03T12:12:57.874+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
